script/release/prepare-release.js

```diff
@@ -182,7 +187,7 @@ async function promptForVersion (version) {
 
 // function to determine if there have been commits to main since the last release
 async function changesToRelease () {
-  const lastCommitWasRelease = new RegExp('^Bump v[0-9.]*(-beta[0-9.]*)?(-nightly[0-9.]*)?$', 'g');
+  const lastCommitWasRelease = new RegExp('^Bump v[0-9.]*(-beta[0-9.]*)?(-alpha[0-9.]*)?(-nightly[0-9.]*)?$', 'g');
```

Comment:
```suggestion
  const lastCommitWasRelease = new RegExp('^Bump v[0-9]+\.[0-9]+\.[0-9]+(-beta\.[0-9]+)?(-alpha\.[0-9]+)?(-nightly\.[0-9]+)?$', 'g');
```

Reply:
Looks like es-lint thinks these aren't needed! Added an eslint-ignore comment to this, but can also revert it